xml/security-barbican.xml

```diff
@@ -197,35 +164,43 @@
     </para>
     <warning>
      <para>
-      Do not change your master key after deploying it to Barbican.
+      Do not change your master key after deploying it to &secret_store;.
      </para>
     </warning>
    </listitem>
    <listitem>
     <para>
      If you decide to make configuration changes to your clean install of
-     &kw-hos-phrase;, you will need to redeploy the Barbican service. For
+     &kw-hos-phrase;, you will need to redeploy the &secret_store; service. For
      details on available customization options, please see
      <xref linkend="barbicanAdmin"/>.
     </para>
    </listitem>
   </itemizedlist>
  </section>
- <section>
-  <title>Error Received if Root Logical Volume is Too Small</title>
+ <section xml:id="sec.barbican.kmip">
+  <title>Master Key Configuration</title>
   <para>
-   Barbican currently supports databases and KMIP as its secret store back-ends
-   In &ostack-bare; upstream additional back-ends are available, such as the
-   PKCS11 and dogtag plug-ins, but they are not tested or supported by
+   &secret_store; currently supports databases and KMIP as its secret store
+   back-ends In &ostack-bare; upstream additional back-ends are available, such
+   as the PKCS11 and dogtag plug-ins, but they are not tested or supported by
    &kw-hos;.
   </para>
   <para>
-   In &kw-hos;, by default Barbican is configured to use a database as a secret
-   (keys) storage back-end. This back-end encrypts Barbican-managed keys with a
-   project level key (kek or key encryption key) before storing it in the
+   In &kw-hos;, by default &secret_store; is configured to use a database as a
+   secret (keys) storage back-end. This back-end encrypts
+   &secret_store;-managed keys with a project level key
+   (<emphasis>KEK</emphasis>/Key Encryption Key) before storing it in the
    database. Project-level keys are encrypted using a master key. As part of
-   the initial Barbican configuration, you must generate and configure this
-   master key.
+   the initial &secret_store; configuration, you must generate and configure
+   this master key.
+  </para>
+  <para>
+   When &secret_store; is used with <literal>simple_crypto_plugin</literal> as
+   its secret store back-end, its master key needs to be defined
+   <emphasis role="bold">before initial deployment</emphasis>. If no key is
+   specified before deployment, the default master key is
+   used&mdash;<emphasis role="bold">this practice is discouraged</emphasis>.
```

Comment:
As style thing: we generally avoid using `<emphasis role="bold">`. Instead, I would recommend creating a note here, and speaking a bit more directly to the user:

```
<important>
  <title>Undefined Master Key</title>
  <para>
    If you do not define a master key before deployment, the default master key is
    used. We recommend against this practice.
  </para>
</important
```

Ideally, we could write something about the consequences, i.e. if this is a security issue or so, but at least I don't know details here.